float32 height
duration time_from_start
---
